qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitSteps.java

```diff
@@ -47,11 +47,11 @@ public void clickOnFindPatientRecordApp() {
     @And("user searches for the patient record")
     public void searchForPatientRecord() {
     	findPatientPage.enterPatient(patient.identifier);
-    	findPatientPage.waitForPageToLoad();
+        findPatientPage.waitForPageToLoad();
     }
     
-    @And("user clicks on the first patient record")
-    public void clickTheFirstPatientRecord() {
+    @And("user clicks on the patient returned in the patient table")
```

Comment:
The method name and above text do not match.

Reply:
Resolved.